packages/assertions/tm-assertions/tm-tmRequired.json

```diff
@@ -0,0 +1,26 @@
+{
+    "title": "tm-tmRequired",
+    "description": "To guarantee the implementation of particular kind of interaction models, Thing Model definitions MUST use the JSON member name tm:required.",
+    "$schema ": "http://json-schema.org/draft/2019-09/schema#",
+    "also": ["tm-tmRequired-JSONPointer", "tm-tmRequired-array"],
+    "is-complex":false,
+    "type": "object",
+    "properties": {
+        "tm:required": {
+            "type": "array",
+            "items": {
+                "$ref": "#/$defs/JSONPointer"
+            }
+        }
+    },
+    "required": [
+        "tm:required"
+    ],
+    "additionalProperties": true,
+    "$defs": {
+        "JSONPointer": {
+            "type": "string",
+            "format": "uri-reference"
```

Comment:
there is even a json-pointer type if I am not wrong. Also this definitely needs multiple examples since the pointer can be a full URL or just an in-document relative pointer

Reply:
Yes, but I thought that the will always use the uri-format (starting with #) and not normal ones (starting with /). If both will be used I will the normal one too.